open-api.yaml

```diff
@@ -892,59 +869,17 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/dumps'
+                $ref: '#/components/responses/202'
               examples:
-                Created:
+                '202':
                   value:
-                    uid: 20210719-114144097
-                    status: in_progress
-                    startedAt: '2021-07-19T14:31:16.920473Z'
+                    uid: 0
+                    indexUid: null
+                    status: enqueued
+                    type: dumpCreation
+                    enqueuedAt: '2021-07-19T14:31:16.920473Z'
         '401':
           $ref: '#/components/responses/401'
-  '/dumps/{dumpUid}/status':
```

Comment:
I think the PR description saying "/dumps/{dumpUid}/status is removed" is missing in the PR description
The PR description helps me a lot when creating issues for the Meilisearch milestones, since it gathers all the breaking changes and the main enhancements 😇